app/helpers/users_helper.rb

```diff
@@ -0,0 +1,2 @@
+module UsersHelper
```

Comment:
Don't use scaffold to create all files.
Only create the required files for now.